_includes/components/header--basic.html

```diff
@@ -0,0 +1,75 @@
+{% if header %}
+{% include banner.html %}
```

Comment:
Looks like the sample headers on the standards site have the banner within the `<header>` element - should this remain within there for consistency?

Reply:
That's a good question, I know there has been talk about making the USA - banner it's own component. I'm fine with moving it back inside the `<header>` element.